content/css/sel4.css

```
@@ -266,4 +266,15 @@ a.divLink {
   color:#B78B41;  /* emphasisDark */
   font-size:15px;
   font-weight:bold;
-}
\ No newline at end of file
+}
+
+.emph {
+    color:#B78B41;  /* emphasisDark */
```

Trying out a different one:
```suggestion
    color:#668B37;  /* greenDark */
```